pay-api/src/pay_api/services/payment_account.py

```diff
@@ -272,22 +273,55 @@ def _save_account(cls, account_request: Dict[str, any], payment_account: Payment
         payment_account.bcol_account = account_request.get('bcolAccountNumber', None)
         payment_account.bcol_user_id = account_request.get('bcolUserId', None)
 
-        billable = get_str_by_path(account_request, 'paymentInfo/billable').lower() == 'true'
+        payment_info = account_request.get('paymentInfo')
+        billable = payment_info.get('billable', True)
         payment_account.billable = billable
 
-        # Create account in CFS based on payment method
-        # If payment method is PAD, OB, EFT, WIRE create a cfs account
-        # For government accounts (billable false) create a cfs account for EJV transactions
+        # Steps to decide on creating CFS Account or updating CFS bank account.
+        # Updating CFS account apart from bank details not in scope now.
+        # Create CFS Account IF:
```

Comment:
will it be any good if we move this logic to the factory class? will that do any good?